tests/model/test_suite_public_name.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from django.db.utils import IntegrityError
+
+SAME_SUITE_PUBLIC_NAME = 'same suite name'
+SAME_TEST_PUBLIC_NAME = 'same test name'
+SAME_SUITE = 'same suite'
+SAME_TEST = 'same test'
+
+
+@pytest.mark.parametrize("suite_public_name_1, suite_public_name_2,"
+                         "test_public_name_1, test_public_name_2,"
+                         "suite_1, suite_2, test_1, test_2", [
```

Comment:
Wouldn't it be better to simply remove all `_1` suffixes?

Reply:
OK, I will remove it.